configure.ac

```diff
@@ -891,17 +909,162 @@ fi
 
 # Updated certificate verification callback (introduced with 2.0.0, not present
 # in 2.0.0-rc4 or earlier)
-if test "x${have_freerdp2}" = "xyes"
+if test "x${have_freerdp}" = "xyes"
 then
     AC_CHECK_MEMBERS([freerdp.VerifyCertificateEx],,,
-                     [[#include <freerdp/freerdp.h>]])
+            [[#include <freerdp/freerdp.h>]])
```

Comment:
There seem to be some alignment issues throughout the `configure.ac` file, now. If we're sticking to how things are generally done throughout the file, the start of this should be aligned with the `[freerdp.VerifyCertificateEx]` line above, but it seems to be less indented, now.. There are several of these occurrences throughout the changes in the file.

Reply:
Updated formatting

Reply:
Should be cleaner now